src/renderer/i18n/ru/swap.ts

```diff
@@ -12,7 +12,9 @@ const swap: SwapMessages = {
   'swap.errors.amount.balanceShouldCoverChainFee':
     'Комиссия транзакции {fee} дожна покрываться вашим балансом (сейчас {balance}).',
   'swap.errors.amount.outputShouldCoverChainFee':
-    'Исходящая комиссия {fee} должна покрываться получаемым количеством (сейчас {amount}).'
+    'Исходящая комиссия {fee} должна покрываться получаемым количеством (сейчас {amount}).',
+  'swap.note.lockedWallet': 'You need to unlock your wallet to swap - RU',
+  'swap.note.nowallet': 'Create / import a wallet to swap - RU'
```

Comment:
`'Create / import a wallet to swap - RU' `-> `'Create or import a wallet to swap - RU'`